SonarQube.TeamBuild.PostProcessor/Resources.resx

```diff
@@ -125,7 +125,7 @@ Analysis config file: {2}
 Please delete the analysis config file and try the build again.</value>
   </data>
   <data name="ERROR_CmdLine_DisallowedArgument" xml:space="preserve">
-    <value>This command line argument is not valid in the "end" phase: {0}</value>
+    <value>This setting is not valid in the "end" phase: {0}. Possible cause: you are not running the latest C# plugin version.</value>
```

Comment:
Changing "setting" to "command line argument" makes sense. I'm not sure about the sentence you'd added, though; it sets the expectation that future versions of the MSBuild Runner will be less restrictive, which may or may not be the case. I'm not sure whether it really helps the user. You could say something more neutral like "This setting is not valid the "end" phase in this version of the MSBuild.SonarQube.Runner: {0}".